spec/controllers/redirections_controller_spec.rb

```diff
@@ -3,25 +3,23 @@
 RSpec.describe RedirectionsController do
   describe "GET :next" do
     it "redirects to the next site" do
-      gabe = create(:redirection)
-      edward = create(:redirection, next: gabe)
-      gabe.update(next: edward)
+      gabe = Redirection.find_by!(slug: "gabe")
```

Comment:
[Yes](https://github.com/gabebw/hotline-webring/blob/master/spec%2Fsupport%2Fdatabase_cleaner.rb#L5). It hides some logic, but there are a lot of things we can assume if there are two redirections in the database, which is the case.